### PR TITLE
Updated format of code

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [ ] New topic
- [X] Content update
- [X] Content fix or rewrite
- [X] Bug fix or improvement

## Summary
Updated code format on Create an integration document.
When this pull request is merged, it will...

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/get-started/create-integration.html
- https://devdocs.magento.com/guides/v2.2/get-started/create-integration.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/get-started/create-integration.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ To develop a module, you must:
    Also create  `etc`, `etc/integration`, and `Setup` subdirectories under `module-<module_name>`, as shown in the following example:
 
     <pre>
```
Comment (jeff-matthews) [inline guides/v2.2/get-started/create-integration.md]:
```suggestion
    ```bash
```

File: guides/v2.3/get-started/create-integration.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ To develop a module, you must:
    Also create  `etc`, `etc/integration`, and `Setup` subdirectories under `module-<module_name>`, as shown in the following example:
 
     <pre>
```
Comment (jeff-matthews) [inline guides/v2.3/get-started/create-integration.md]:
```suggestion
    ```bash
```

File: guides/v2.3/get-started/create-integration.md [hunk 3]
```diff
@@ -37,9 +37,9 @@ To develop a module, you must:
...
-    cd &lt;magento_base_dir>
-    mkdir -p vendor/&lt;vendor_name>/module-&lt;module_name>/etc/integration
-    mkdir -p vendor/&lt;vendor_name>/module-&lt;module_name>/Setup
+    cd <magento_base_dir>
+    mkdir -p vendor/<vendor_name>/module-<module_name>/etc/integration
+    mkdir -p vendor/<vendor_name>/module-<module_name>/Setup
    </pre>
```
Comment (jeff-matthews) [inline guides/v2.3/get-started/create-integration.md]:
```suggestion
    ```
```

File: guides/v2.2/get-started/create-integration.md [hunk 4]
```diff
@@ -37,9 +37,9 @@ To develop a module, you must:
...
-    cd &lt;magento_base_dir>
-    mkdir -p vendor/&lt;vendor_name>/module-&lt;module_name>/etc/integration
-    mkdir -p vendor/&lt;vendor_name>/module-&lt;module_name>/Setup
+    cd <magento_base_dir>
+    mkdir -p vendor/<vendor_name>/module-<module_name>/etc/integration
+    mkdir -p vendor/<vendor_name>/module-<module_name>/Setup
    </pre>
```
Comment (jeff-matthews) [inline guides/v2.2/get-started/create-integration.md]:
```suggestion
    ```
```

**Comment by gelanivishal** [comment]
@jeff-matthews I have updated your suggestion.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**